test/integration/request-body/protocolProfileBehavior.test.js

```diff
@@ -4,6 +4,7 @@ var net = require('net'),
 describe('protocolProfileBehavior', function () {
     var server,
```

Comment:
Add an `afterEach` block inside this describe block to set `rawRequest` to `null`.